Microsoft.Toolkit.Uwp.SampleApp/Microsoft.Toolkit.Uwp.SampleApp.csproj

```diff
@@ -145,7 +145,7 @@
   </ItemGroup>
   <ItemGroup>
     <!-- A reference to the entire .Net Framework and Windows SDK are automatically included -->
-    <Content Include="Assets\Helpers.png" />
+    <Content Include="Assets\Helpers_old.png" />
```

Comment:
What's old about this now? ;)

Can it be removed?

Reply:
Yes, will removed